packages/console/src/components/ItemPreview/index.module.scss

```diff
@@ -9,15 +9,23 @@
     margin-left: _.unit(3);
   }
 
-  .title {
```

Comment:
Where is the usage of the new class name `.content`? Forgot to check in .tsx file change?

Reply:
This class name is only used in `compact` mode before.